3_navigation.ipynb

```diff
@@ -121,15 +121,15 @@
     "lang": "en"
    },
    "source": [
-    "Let's check the information by turning the check on and off in \"Displays\" panel. The meaning of each item is as follows:\n",
+    "Let's check the information by turning the checks on and off in the \"Displays\" panel. The meaning of each item is as follows:\n",
     "\n",
     "- Map: Predefined map information.\n",
     "\n",
     "- global_cost_map: Cost map generated based on the predefined map.\n",
     "\n",
     "- local_cost_map: Cost map generated based on laser scanner information.\n",
     "\n",
-    "Here, the cost map is a map that numerically expresses areas that are likely to pass or with obstacles, and is used for autonomous movement."
+    "Here, the cost map is a map that numerically expresses areas where the robot is like to pass through or areas with obstacles, and is used for autonomous movement."
```

Comment:
"where the robot is like" should really be "where the robot is likely".
Sorry, it is possible that I missed the "-ly" in the original comment. :(